bazel/WORKSPACE

```diff
@@ -24,11 +24,11 @@ tf_configure(
 new_http_archive(
     name = "ngraph",
     build_file = "bazel/ngraph.BUILD",
-    sha256 = "741812c14c70882e1ddb28e27c4a5e890b5fef3b2ec81ab6662dcd47127e9a35",
-    strip_prefix = "ngraph-d3453447883fe48e63126316d5da72fcbd47dded",
+    sha256 = "ec8c8933d7e6f654cf5b316475f67b6934c019f00144e9999f9b4877a53b2a1f",
+    strip_prefix = "ngraph-df9e23761f5f7a7a7d0134c8ae0c92a8bad2e961",
```

Comment:
This should be set to release tag i.e., `v0.15.0-rc.1` at this point. I will add this change.